### PR TITLE
Fixes, small improvements

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: weblate_web/templates/donate/form.html [hunk 1]
```diff
@@ -71,7 +71,7 @@ <h1 class="section-title min-m">{% trans "Your donation" %}</h1>
                 </form>
             </div>
             <p class="small">
-                {% trans "Please note — a full invoice for your contribution can be issued if need be." %}
+                {% trans "Please note—a full invoice for your contribution can be issued if need be." %}
```
Comment (comradekingu) [inline weblate_web/templates/donate/form.html]: this is not donation

File: weblate_web/templates/donate.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 {% load static %}
 {% load i18n %}
 {% block title %}{% trans "Donate to Weblate" %}{% endblock %}
-{% block description %}{% blocktrans %}Weblate is provided free of charge and welcomes donations. There are many ways to contribute. Discuss it with your associates.{% endblocktrans %}{% endblock %}
+{% block description %}{% blocktrans %}Weblate does not require payment, and welcomes donations. There are many ways to contribute. Discuss it with your associates.{% endblocktrans %}{% endblock %}
```
Comment (comradekingu) [inline weblate_web/templates/donate.html]: What about "and welcomes contributions"?

Comment (nijel) [reply weblate_web/templates/donate.html]: As this is on the donate page, I'd probably stick with donations here.

**Comment by nijel** [comment]
Merged, thanks for your contribution!

File: weblate_web/templates/donate/form.html (hunk 1 above)
Comment (nijel) [reply weblate_web/templates/donate/form.html]: Changed in 7ef7835a

**Pull request merged**